manifests/outputs/kafkaoutput.pp

```diff
@@ -0,0 +1,142 @@
+# Output plugin that connects to a Kafka broker and sends messages to the specified topic
+#
+# === Parameters:
+#
+# $ensure::                      	This is used to set the status of the config file: present or absent
+#
+# $message_matcher::             	Boolean expression, when evaluated to true passes the message to the filter for processing.
```

Comment:
Same request as for kafkainput. Send the reader for common parameters definition to tcpoutput module